misk/src/main/kotlin/misk/web/interceptors/TracingInterceptor.kt

```diff
@@ -28,9 +29,22 @@ internal class TracingInterceptor internal constructor(private val tracer: Trace
   }
 
   override fun intercept(chain: Chain): Any? {
-    var result: Any? = null
+    val result: Any?
+    var scope: Scope? = null
 
-    tracer.buildSpan("${chain.action}").startActive(true).use { result = chain.proceed(chain.args) }
+    try {
+      scope = tracer.buildSpan("${chain.action}").startActive(true)
+      result = chain.proceed(chain.args)
+      if (result is Response<*> && result.statusCode / 100 > 3) {
+        Tags.HTTP_STATUS.set(scope.span(), result.statusCode)
```

Comment:
set the ``HTTP_STATUS`` tag for non-errors as well (e.g. 2XX / 3XX) ?

Reply:
No good reason not to, to be honest. I'll add it and if it gets annoying we can always just remove it.